main.js

```diff
@@ -0,0 +1,24 @@
+// let els = document.querySelectorAll("a");
+// for(let el of els){
+//     if(el.textContent.includes("All Posts")){
+//         el.remove();
+//     }
+// }
+
+let el = document.querySelector("header nav ul li");
+el.remove();
+
+let els = document.querySelectorAll("span");
+for(let el of els){
+    if(el.textContent.includes("Read Time:")){
+        el.remove();
+    }
+}
+
+let removePost = document.querySelector("main .posts article");
+removePost.remove();
```

Comment:
```suggestion
let articles = document.querySelectorAll(".posts article");
let lastArticle = articles[articles.length - 1]
lastArticle.remove();
```
Although the tests are passing, you actually aren't removing the **_last_** article successfully. Instead, you removed the **_first_** article, because `querySelector` will grab the first match in the file. To get the **_last_** one, you need to use `querySelectorAll`, which returns a node list. Then, grab the last element in the node list and .remove it.